src/view/renderer.js

```diff
@@ -440,10 +440,13 @@ export default class Renderer {
 		const domConverter = this.domConverter;
 		const domElement = domConverter.getCorrespondingDom( viewElement );
 
-		if ( !domElement ) {
-			// If there is no `domElement` it means that it was already removed from DOM.
-			// There is no need to update it. It will be updated when re-inserted.
-			return;
+		// Refresh children of this `viewElement` if they are already bound with DOM elements. Keep in mind if view
+		// element is bound with DOM element, the DOM element is not regenerated by `DomConverter`.
+		// Sometimes such elements are outdated. For that reason we have to refresh them.
+		for ( let viewChild of viewElement.getChildren() ) {
+			if ( viewChild.is( 'element' ) && domConverter.getCorrespondingDom( viewChild ) ) {
+				this._updateChildren( viewChild, options );
```

Comment:
I would prefer to avoid recursion and do it straight: unbind all children on removing. This solution, even if more efficient, is a little bit hacky and we will have enough hacky code it there renderer anyway, so as long we can make something simpler we should do.